app/helpers/products_helper.rb

```diff
@@ -47,13 +47,15 @@ def set_sorting(test, test_status)
 
     case test_status
     when 'passing'
-      return 5
+      return 6
```

Comment:
These numbers are set on a cell to dictate how DataTables sorts a given column... maybe I'm just lacking imagination but how might replacing these with constants work?